modules/mvt/src/lib/parse-mvt.js

```diff
@@ -41,7 +41,14 @@ export default function parseMVT(input, options) {
     }
   });
 
-  return options.mvt._format === 'binary' ? geojsonToBinary(features) : features;
+  if (options.mvt._format === 'binary') {
+    const data = geojsonToBinary(features);
+    // TODO decide where to store this
+    // @ts-ignore
+    data.byteLength = arrayBuffer.byteLength;
```

Comment:
Is this accurate though? I'd expect the decoded binary to be larger than the compressed form.

Reply:
Probably a good-enough approximation for now

Reply:
Yes intended an approximation only. I'll add a comment in another PR.